python_modules/libraries/dagster-dask/dagster_dask_tests/test_execute.py

```diff
@@ -244,6 +244,9 @@ def test_dask_terminate():
         assert DagsterEventType.PIPELINE_FAILURE in result_types
 
 
+@pytest.mark.skip(
+    "Failing with RuntimeError: This event loop is already running since distributed==2022.1.0"
+)
```

Comment:
i think its just a problem with the invocation down on line 274 - must need a different call since the event loop is already running

Reply:
I couldn't find an obvious candidate, but I don't have much asyncio experience